Decisions.Microsoft365.Exchange/InputExchangeSettings.cs

```diff
@@ -0,0 +1,55 @@
+using System.ComponentModel;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.Design.Properties;
+using DecisionsFramework.Design.Properties.Attributes;
+
+namespace Decisions.Microsoft365.Exchange;
+
+public class InputExchangeSettings: ICreateChildEntityInFolder
```

Comment:
I'm really confused. I think you've branched off the wrong place or something. What are these changes?

Reply:
This is the new input to the duplicate steps (CalendarSteps -> ExchangeCalendarSteps and so on)
That what we were discussing earlier. SInce ExchangeSettings inherits AbstractModuleSettings, there can be only one instance of it per project. So instead of using ExchangeSettings, we introduced InputExchangeSettings and pass it to the methods(steps) instead of passing ExchangeSettings. 

tagging @noah-decisions if I missed more context.

Reply:
Yes, this is the new object to use for the settings override input to the steps, replacing where we used the modules settings object before